src/ConnectedMode/Hotspots/IHotspotDocumentClosedHandler.cs

```diff
@@ -0,0 +1,28 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarLint.VisualStudio.ConnectedMode.Hotspots
+{
+    /// <summary>
+    /// Marker interface for the component that listens for documents being closed
+    /// and triggers the removal of issues from the local hotsp
+    /// </summary>
+    public interface IHotspotDocumentClosedHandler { }
```

Comment:
New interface in the ConnectedMode project.

I couldn't move the hotspot store etc to the ConnectedMode project without moving a whole load of other classes, so I created this interface instead.